app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_experiment_serializer.py

```diff
@@ -968,7 +968,7 @@ def test_cant_review_for_requesting_user(self):
         self.assertFalse(serializer.is_valid(), serializer.errors)
         self.assertIn("publish_status", serializer.errors)
 
-    def test_can_review_for_requesting_user_when_idle(self):
+    def test_cannot_approve_for_requesting_user_when_idle(self):
```

Comment:
It's not even really that, it's that Idle > Approved isn't even an allowed transition.  I'd nuke this particular test altogether and just make sure the allowed transition tests are parameterized to cover all the transitions.